ChangeLog.md

```diff
@@ -1,4 +1,13 @@
-# Minicraft+ version 2.2.0
+# Minicraft+ Version 2 Changelog
+
+This changelog is related to all the Minicraft+ versions.
+All notable changes to this project will be documented in this file.
+
+## [2.2]
```

Comment:
These version numbers do not correspond with https://keepachangelog.com/en/1.0.0/.

Reply:
As what I have said:
> In order to make it more tidier, I copy the previous txt file contents into /docs and reduce the md file from the begin with 2.0.0 version changelog. 
The format I made is to show that its versioning system has been changed. 
I also temped to make it little similar to the format the wiki looks like.

Reply:
That does not explain why you chose not to adhere to the standard. It just looks ugly now.

Reply:
In semantic versioning system, the major, minor and patch could be separated into multiple sections.

Reply:
Just make it simpler. Stop over-complicating things.

Reply:
Alright.